src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li');
+const empList = document.querySelector('ul');
```

Comment:
You could create only one variable 'empList' (strange name) and use it like [... empList.children] instead of creating another variable 'employees'